fuse-core/src/main/java/com/curefit/service/external/LISLabOrderService.java

```
@@ -59,11 +62,15 @@ private LisPatientBookingRequest getLabOrderRequest(DiagnosticOrderRequest diagn
         //lisPatientBookingRequest.setFatherHusband(diagnosticOrderRequest.getUser().getFatherHusband());
         lisPatientBookingRequest.setMobileNumber(userDetails.getMobile());
         lisPatientBookingRequest.setPinCode(Long.parseLong(userDetails.getPincode()));
-        lisPatientBookingRequest.setCentreId(
-            diagnosticOrderRequest.getInCentreDiagnosticRequest().getCentreId().toString());
-        lisPatientBookingRequest.setBookingId(diagnosticOrderRequest.getCarefitOrderId());
+        if (diagnosticOrderRequest.getInCentreDiagnosticRequest() != null) {
```

Here we should check if the 2 centre id match or not. If they don't then let's throw an exception. Also, we should check this in the top level service only. Because it will hold true for any type of labs.